sycl/doc/extensions/template.asciidoc

```diff
@@ -96,6 +96,8 @@ This extension is no longer implemented in {dpcpp}.  This specification is
 being archived only for historical purposes.  *The APIs defined in this
 specification no longer exist and cannot be used.*
 
+== Backend support status and portability recommendation
```

Comment:
We should be clear about when an extension needs to include this section.  Presumably, it is only needed if the extension has some limitation w.r.t. backends?

I think I like your previous section title better "Backend support status".

Reply:
OK I've reverted to the previous title and added the italics that it the section is only necessary if there is a backend limitation.